extras/ams_wrapper/tests/accuracy/vehicle_detection_accuracy.py

```diff
@@ -0,0 +1,146 @@
+    channel = grpc.insecure_channel("{}:{}".format("localhost","9000"))
+    stub = prediction_service_pb2_grpc.PredictionServiceStub(channel)
+    #img = load_tf_image(file_path).reshape(1,3,384, 672)
+    # when the same preprocessing is used, results would be identical
+    img = load_cv_image(file_path).reshape(1,3,384, 672)
+    request = predict_pb2.PredictRequest()
+    request.model_spec.name = "vehicle_detection_adas"
+    request.inputs["data"].CopyFrom(make_tensor_proto(img, shape=(img.shape)))
+    result = stub.Predict(request, 10.0)
+    output = make_ndarray(result.outputs["detection_out"])
+    return output
+
+def interpret_array(output, img_out):
+    results = np.zeros((200,5),np.float32)
+    for i in range(0, 199):  # there is returned 200 detections for each image in the batch
+        detection = output[:,:,i,:]
+        # each detection has shape 1,1,7 where last dimension represent:
+        # image_id - ID of the image in the batch
+        # label - predicted class ID
+        # conf - confidence for the predicted class
+        # (x_min, y_min) - coordinates of the top left bounding box corner
+        #(x_max, y_max) - coordinates of the bottom right bounding box corner.
+
+        x_min = detection[0,0,3]
+        y_min = detection[0,0,4]
+        x_max = detection[0,0,5]
+        y_max = detection[0,0,6]
+        results[[i],:] = [detection[0,0,2],x_min,y_min,x_max-x_min,y_max-y_min]
+        if detection[0,0,2] > 0.5:  # ignore detections confidence <0.5
+            print("detection", i )
+            x_min_s = int(detection[0,0,3]*672)
+            y_min_s = int(detection[0,0,4]*384)
+            x_max_s = int(detection[0,0,5]*672)
+            y_max_s = int(detection[0,0,6]*384)
+        # box coordinates are proportional to the image size
+            print("confidence", detection[0,0,2])
+            print("x_min", x_min, x_min_s )
+            print("y_min", y_min, y_min_s)
+            print("x_max", x_max,x_max_s)
+            print("y_max", y_max, y_max_s)
+            img_out = cv2.rectangle(img_out,(x_min_s,y_min_s),(x_max_s,y_max_s),(0,0,255),1)
+            # draw each detected box on the input image
+    cv2.imwrite("results_ovms.jpeg",img_out)
+    return results
+
+def interpret_ams(json_txt, img_out):
+    y = json.loads(json_txt)
+    results = np.zeros((200,5),np.float32)
+    detections= y["entities"]
+    i=0
+    for detection in detections:
+        confidence=detection["tag"]["confidence"]
+        print("confidence",confidence)
+        box = detection["box"]
+        print("l:", box["l"]," t:" ,box["t"]," w:",box["w"], " h:", box["h"])
+        if detection["tag"]["confidence"]> 0.5:
+            print(box["l"]+box["w"])
+            img_out = cv2.rectangle(img_out,
+                                    (int(box["l"]*672),int(box["t"]*384)),
+                                    (int((box["l"]+box["w"])*672),int((box["t"]+box["h"])*384)),
+                                     (255,0,0),1)
```

Comment:
As mentioned before, threshold and image height and width could be moved to constants.